hello-world-master/source/components/scene/descriptionscreen.brs

```diff
@@ -0,0 +1,10 @@
+sub init()
+    m.poster = m.top.findNode("itemPoster")
+    m.label = m.top.findNode("description")
+end sub
+
+sub showcontent(event)
+ data = event.getData()
+ m.poster.uri = data.image_1080_url
+ m.label.text = data.description
```

Comment:
If you check above, you used TAB for indentation and here only 1 space. To keep consistency please use 1TAB for indentation